eslzArm/managementGroupTemplates/policyDefinitions/policies.json

```diff
@@ -9535,8 +9535,14 @@
                   "existenceCondition": {
                     "allOf": [
                       {
-                        "field": "Microsoft.Sql/servers/databases/vulnerabilityAssessments/recurringScans.emails",
-                        "equals": "[[parameters('vulnerabilityAssessmentsEmail')]"
+                        "count": {
+                          "field": "Microsoft.Sql/servers/databases/vulnerabilityAssessments/recurringScans.emails[*]",
+                          "where": {
+                            "field": "Microsoft.Sql/servers/databases/vulnerabilityAssessments/recurringScans.emails[*]",
+                            "in": "[[parameters('vulnerabilityAssessmentsEmails')]"
```

Comment:
Am I correct in thinking that this checks that the e-mail addresses configured already in ARM are actually in the array and therefore theres no chance that there could be say 3 email addresses supplied in the array on assignment but actually 3 different email addresses are already configured and therefore the policy doesn't flag as non-compliant?

Reply:
Correct. The design behavior is as such. We have three mandatory email addresses that must exist. Someone has configured five email addresses for VA alerts. This will loop through all five email addresses and ensure that the three mandatory ones are present. We don't care about the additional two and there's really no harm in them being there.

Reply:
And I'll review the existing built-ins and see if we have any that can be pulled in to address these issues. Additionally, I'll take a look at issue #695.

Reply:
> Correct. The design behavior is as such. We have three mandatory email addresses that must exist. Someone has configured five email addresses for VA alerts. This will loop through all five email addresses and ensure that the three mandatory ones are present. We don't care about the additional two and there's really no harm in them being there.

Does this not mean the "equals" should actually be "greaterOrEquals"?

Reply:
It could be that, but it wouldn't change the effective logic of that loop. I'm just saying that I must have x number of matches in the email addresses array within ARM, where x equals the length of my email addresses parameter in the Policy. There can't be more matches than there are elements in my email addresses Policy parameter. Keep in mind that you can't put the same email address in that field within ARM twice. I hope that explanation makes sense. :)